mta-sts.md

```diff
@@ -51,11 +51,11 @@
 
 .# Abstract
 
-SMTP MTA-STS is a mechanism enabling mail service providers to declare their
-ability to receive TLS-secured connections, an expectated validity of
-certificates presented by their MX hosts, and to request that sending SMTP
-servers report upon and/or refuse to deliver messages that cannot be delivered
-securely.
+SMTP Mail Transfer Agent Strict Transport Security (SMTP STS) is a mechanism
+enabling mail service providers to declare their ability to receive TLS-secured
+connections, an expectated validity of certificates presented by their MX hosts,
+and to request that sending SMTP servers report upon and/or refuse to deliver
+messages that cannot be delivered securely.
```

Comment:
Do we need to remove the bit about requesting reports, as that's largely part of TLSRPT now?